Services/BrokerService/BrokerService.csproj.user

```diff
@@ -4,6 +4,6 @@
     <DebuggerFlavor>ProjectDebugger</DebuggerFlavor>
   </PropertyGroup>
   <PropertyGroup>
-    <ActiveDebugProfile>BrokerServices</ActiveDebugProfile>
+    <ActiveDebugProfile>IIS Express</ActiveDebugProfile>

```

Comment:
Interesting why you changed this setting, but it's not really important))

Reply:
Done